tensorboard/webapp/metrics/internal_types.ts

```diff
@@ -100,7 +100,7 @@ export interface HeaderEditInfo {
 }
 
 export interface HeaderToggleInfo {
-  headerType: ColumnHeaderType;
+  header: ColumnHeader;
```

Comment:
I probably would have just changed this to use headerName. It would mostly have helped with tests. I am totally fine with this though.

Reply:
That makes sense, I prefer this approach though in case we end up changing our strategy for verifying uniqueness again.